README.md

```diff
@@ -1,44 +1,85 @@
-# Quick start
+# Shortest
+**[shortest.com](https://shortest.com)**
 
-1. Install the Shortest package
-   
-```
+AI-powered natural language end-to-end testing framework.
+
+## Features
+- Natural language E2E testing framework
+- AI-powered test execution using Anthropic Claude API
+- Built on Playwright
+- GitHub integration with 2FA support
+
+## Installation
+```bash
 npm install @antiwork/shortest
+# or
+pnpm add @antiwork/shortest
+# or
+yarn add @antiwork/shortest
 ```
 
-2. Write your first test
+## Quick Start
 
-```
-import { define } from 'shortest'
-
-define('Login Flow', () => {
-  test('User can login')
-    .given('I am on the login page')
-    .when('I enter my credentials')
-    .expect('I should be logged in')
-})
-```
+1. Determine your test entry and add your Anthropic API key in config file: `shortest.config.ts`
 
-3. Don't forget to add an API key:
+```typescript
+import type { ShortestConfig } from '@antiwork/shortest';
 
+export default {
+  headless: false,
+  baseUrl: 'http://localhost:3000',
+  testDir: 'app/__tests__',
+  anthropicKey: process.env.ANTHROPIC_API_KEY
+} satisfies ShortestConfig; 
 ```
-ANTHROPIC_API_KEY=your_api_key
-```
 
-4. Run your tests with:
+2. Write your test in your test directory: `app/__tests__/login.test.ts`
+
+```typescript
+import { test } from '@antiwork/shortest'
+
+const loginCredentials = {
+  email: process.env.EMAIL,
+  password: process.env.PASSWORD,
+}
 
+test('Login to the app using email and password', githubCredentials)
```

Comment:
Can we just use the dictionary here? No real point in creating it imo, and adds lines to the example.

Reply:
Well actually we can just have plain email and password in the test description. It's cleaner. Should I do that?

Reply:
I'm open to supporting both, I agree that inline is probably easier/simpler though.

Reply:
Sure I added the dictionary way btw. Let me know if there is something else.